google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITTransferManagerTest.java

```diff
@@ -253,6 +253,23 @@ public void uploadNonexistentFile() throws Exception {
     }
   }
 
+  @Test
+  public void downloadNonexistentBucket() throws Exception {
+    TransferManagerConfig config =
+        TransferManagerConfigTestingInstances.defaults(storage.getOptions());
+    try (TransferManager transferManager = config.getService()) {
+      String bucketName = "this-bucket-does-not-exist";
```

Comment:
Maybe include a uuid in this name, to prevent the case of someone creating this bucket and breaking our tests.